bin/waagent2.0

```diff
@@ -3457,9 +3457,21 @@ class RdmaHandler(object):
         """
         Write config string to /dev/hvnd_rdma
         """
-        Log("Set /dev/hvnd_rdma")
-        self.wait_rdma_dev()
-        self.write_rdma_dev_conf()
+        skip_rdma_device = False
```

Comment:
So every time I do a "setup.py install", this file ended up getting installed to /usr/sbin/waagent2.0 on my test machine. I have to change it to make my system work. If this change is not necessary on 2.1.X, we can ignore this change.